app/uk/gov/hmrc/agentclientrelationshipsfrontend/views/client/ConsentInformationPage.scala.html

```diff
@@ -44,30 +59,28 @@
 ) {
 
   <h2 class="govuk-caption-xl">@messages(serviceKey)</h2>
-  <h1 class="govuk-heading-xl">@pageTitle</h1>
-  <p class="govuk-body">@Html(messages(s"$key.p1", agentName, agentRole, taxServiceName))</p>
+  <h1 class="govuk-heading-xl">@Html(pageTitle)</h1>
 
-  @if(appConfig.emaEnabled && messages.isDefinedAt(s"$key.$serviceKey.p2")) {
-    <p class="govuk-body">@messages(s"$key.$serviceKey.p2", taxServiceName)</p>
-    <p class="govuk-body">@messages(s"$key.$serviceKey.p3")</p>
-    <ul class="govuk-list govuk-list--bullet">
-      <li>@messages(s"$key.$serviceKey.li1")</li>
-      <li>@messages(s"$key.$serviceKey.li2")</li>
-    </ul>
-    <p class="govuk-body">@Html(messages(s"$key.$serviceKey.guidance", appConfig.guidanceUrlForAgentRoles))</p>
+  @if(agentRoleKey == "mainAgent" || agentRoleKey == "suppAgent") {
+      <h2 class="govuk-heading-l">@messages(s"$key.section1.h2")</h2>
+      <p class="govuk-body">@messages(s"$key.section1.p1", taxServiceName)</p>
+      <p class="govuk-body">@messages(s"$key.section1.p2")</p>
+      @warningHtml
+      <p class="govuk-body">@Html(messages(s"$key.section1.p3", appConfig.guidanceUrlForAgentRoles))</p>
   }
```

Comment:
```suggestion
  } else { 
    @warningHtml 
  }
```

Reply:
we want to tell non-ITSA users about the replacement of their existing agent (as we do on the consent confirmation page - which is supposed to be removed now also)